javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentStarterImpl.java

```diff
@@ -72,13 +72,22 @@ public void start() {
   }
 
   private void internalStart() {
-    Iterator<LoggingCustomizer> loggingCustomizers =
-        ServiceLoader.load(LoggingCustomizer.class).iterator();
-    LoggingCustomizer loggingCustomizer;
-    if (loggingCustomizers.hasNext()) {
-      loggingCustomizer = loggingCustomizers.next();
-    } else {
-      loggingCustomizer = NoopLoggingCustomizer.INSTANCE;
+    String loggerImplementationName = ConfigPropertiesUtil.getString("otel.javaagent.logging");
+    // default to the built-in stderr slf4j-simple logger
+    if (loggerImplementationName == null) {
+      loggerImplementationName = "simple";
+    }
+
+    LoggingCustomizer loggingCustomizer = null;
+    for (LoggingCustomizer customizer : ServiceLoader.load(LoggingCustomizer.class)) {
+      if (customizer.name().equalsIgnoreCase(loggerImplementationName)) {
+        loggingCustomizer = customizer;
+        break;
+      }
+    }
+    // unsupported logger implementation; defaulting to noop
+    if (loggingCustomizer == null) {
+      loggingCustomizer = new NoopLoggingCustomizer();
     }
```

Comment:
Could shrink this down:

```suggestion
   LoggingCustomizer loggingCustomizer = ServiceLoader.load(LoggingCustomizer.class).stream()
      .findFirst(customizer -> customizer.name().equalsIgnoreCase(loggerImplementationName))
      .orElse(new NoopLoggingCustomizer());
```